frontend/src/store/modules/website/actions.js

```diff
@@ -118,5 +129,23 @@ export default {
     },
     [RESET_DATA]: (context) => {
         context.commit(RESET_WEBSITE_DATA);
-    }
+    },
+
+    [CHANGE_SELECTED_WEBSITE]: (context, payload) => {
+        if (context.state.relateUserWebsites.selectedWebsite.value === payload.value) {
+            return;
+        }
+
+        context.commit(SET_SELECTED_WEBSITE, payload);
+        context.dispatch(FETCH_CURRENT_WEBSITE);
```

Comment:
If you want chain promise to make some actions on error on success, action should return result from dispatch